### PR TITLE
Display member info

### PR DESCRIPTION
**Pull request by thesuss**
PT Story:

### CODE REVIEW

File: app/views/partials/_top.html.haml [hunk 1]
```diff
@@ -16,15 +16,19 @@
...
             %li
               = link_to 'Hem', 'http://sverigeshundforetagare.se/'
-            %li.page_item.page-item-11.current_page_item
+            %li
               = link_to 'Om', 'http://sverigeshundforetagare.se/'
-            %li.page_item.page-item-2
+            %li
               = link_to 'Mer', 'http://sverigeshundforetagare.se/'
-            %li.page_item.page-item-2
+            - if current_user.try(:membership_applications).try(:any?) && current_user.try(:membership_applications).try(:last).try(:persisted?)
```
[the remark this answers is not in the document]
Comment (weedySeaDragon) [reply app/views/partials/_top.html.haml:23]: Should make this a helper method

[the remark this answers is not in the document]
Comment (thesuss) [reply app/views/partials/_top.html.haml:23]: Feel free :D

**Comment by weedySeaDragon** [comment]
This is looking good!
Need a feature test just to be sure the right users can/can't see the pages.

And at some point, the code in the nav menu needs to be shortened/refactored.  (That could easily be a separate chore, IMHO)

**Comment by thesuss** [comment]
There is a "fix the navigation" chore..

**Pull request merged**